examples/visualizations/vis_tree_subdevision.py

```diff
@@ -0,0 +1,62 @@
+"""
+=================================================
```

Comment:
Length of ==== should be the same as title, otherwise Sphinx might fail to produce the example gallery